monai/data/image_reader.py

```diff
@@ -34,6 +34,9 @@
 )
 from monai.utils import MetaKeys, SpaceKeys, TraceKeys, ensure_tuple, optional_import, require_pkg
 
+# workaround for https://github.com/Project-MONAI/MONAI/pull/8062
```

Comment:
```suggestion
# workaround for https://github.com/Project-MONAI/MONAI/issues/8061
```
Should reference the issue? Odd that something like this would come up without change, so long as this doesn't cover up anything else we're fine.

Reply:
Yes, my bad. Thanks for pointing out!